app/src/parser_impl.c

```diff
@@ -112,6 +112,11 @@ parser_error_t _read(parser_context_t *ctx, parser_tx_t *v) {
     CHECK_ERROR(readTransactionVersion(ctx, &v->transactionVersion));
     CHECK_ERROR(readUint64(ctx, &v->spends.elements));
     CHECK_ERROR(readUint64(ctx, &v->outputs.elements));
+    // Check number of outputs is <= than the number of supported outputs in the render mask
```

Comment:
nice trick to use a u64 to mask the renderable items/outputs. This poses the limitation of being able to render up to 64 elements only. is this acceptable ??

Reply:
Yes, it limits the max number of inputs. But I guess we will need something like this, because, once we reach the function to get the num items we need to know the number outputs we will show. 
But maybe we can confirm this is acceptable.